lib/archethic/p2p/message.ex

```diff
@@ -239,7 +246,9 @@ defmodule Archethic.P2P.Message do
         },
         confirmed_validation_nodes: confirmed_validation_nodes
       }) do
-    nb_validation_nodes = length(chain_replication_tree)
+    encoded_nb_validation_nodes_length = length(chain_replication_tree) |> VarInt.from_value()
```

Comment:
As this array cannot overpass a length of 200 we don't need to use VarInt there

Reply:
Changed for length(chain_replication_tree)